tests/behat/features/ra_bug-pieter.feature

```diff
@@ -0,0 +1,33 @@
+Feature: A RA manages tokens tokens registered in the selfservice portal
+  In order to manage tokens
+  As a RA
+  I must be able to manage second factor tokens in RA
+
+  Scenario: Provision an institution and a user to promote later on by an authorized institution
+    Given institution "institution-a.example.com" can "select_raa" from institution "institution-a.example.com"
+      And institution "institution-a.example.com" can "use_ra" from institution "institution-a.example.com"
+      And institution "institution-d.example.com" can "use_raa" from institution "institution-a.example.com"
```

Comment:
Maybe move this entry to the place where inst-d is configured? That makes it easier to follow how individual institutions are configured.

Reply:
This doesn't seem the latest version that's reviewed?